### DISCUSSION TITLE
Week 9 - Project Update

### DISCUSSION THREAD

**Discussion by coopsand** [USU-ITLS-x265-23-summer/class-repo#217] [General]
Hello Everyone!

I forgot to post for Week 9, so here I am now. I have been working on my project and I will say - it took me a while to actually get it up and running. I think my Visual Studio was glitching out a little bit and I ran into issues with setting up the codespace. Once I got that set up, then it was full steam ahead!

Here's the current status of my project:
<img width="848" alt="Screenshot 2023-07-27 at 6 57 15 PM" src="https://github.com/USU-ITLS-x265-23-summer/class-repo/assets/133150498/d2c48fd1-3d5f-4b78-8384-b4d93321430c">

The next tasks I am going to be focusing on are filling in my UX Design and Graphic Design pages. They are pretty empty as they are right now.

For my content, I am going to be showcasing the following:
- An Image of Me
- 1 or 2 UX Design Samples (Images / Links)
- ~5 Graphic Design Pieces

One thing I am wondering about is how I should include JavaScript into this piece, if I should even include it at all? I don't really understand it, or where to apply it.

Hope you are all having a good week!

**Comment by coopsand** [comment]
Additional Comments (as per the rubric item on this week's assignment):
1. https://github.com/orgs/USU-ITLS-x265-23-summer/discussions/189
2. https://github.com/orgs/USU-ITLS-x265-23-summer/discussions/179

**Comment by CJ3315** [comment]
Great work! I have used very little JavaScript on my portfolio, however where I have used it I think that it improves the dynamic functionality of the website. You could do some stylistic additions by using if/else commands when hovering or clicking. Up to you! Looks great as is!

**Reply by coopsand** [reply]
I will have to take a look at your project and see if how you used it will spark any ideas!!

**Comment by kassyoveson** [comment]
I used a very basic java script in my project. I wrote the code incase in the future I decide to go live with the webpage, I used it so my page required information to be put into each box or it won't submit. Your project is coming along nicely.
